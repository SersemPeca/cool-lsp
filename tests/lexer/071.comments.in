This comment was never started *)
