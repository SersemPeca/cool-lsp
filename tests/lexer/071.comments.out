#1 TYPEID This
#1 OBJECTID comment
#1 OBJECTID was
#1 OBJECTID never
#1 OBJECTID started
#1 ERROR: Unmatched *)
